src/Events/Custom_Tables/V1/WP_Query/Modifiers/Events_Only_Modifier.php

```diff
@@ -46,6 +46,26 @@ public function applies_to( WP_Query $query = null ) {
 	 */
 	public function hook() {
 		add_filter( 'posts_pre_query', [ $this, 'filter_posts_pre_query' ], 100, 2 );
+		add_filter( 'tec_events_custom_tables_v1_events_only_modifier_before_get_posts', [ $this, 'filter_ct_query' ]  );
+	}
+
+	/**
+	 * Applies any necessary alterations on Custom_Tables_Query before it runs get_posts();
+	 *
+	 * @since TBD
+	 *
+	 * @param Custom_Tables_Query $query A reference to the query object used to get_posts() for Custom Table queries.
+	 *
+	 * @return Custom_Tables_Query The modified query object.
+	 */
+	public function filter_ct_query( Custom_Tables_Query $query ): Custom_Tables_Query {
+		// These query vars conflict, and will result in unintended results. See TEC-4695.
+		if ( ! isset( $query->query['term'] )
```

Comment:
Skip the `isset( $query->query['term'])` just unset.

Reply:
Should the `$query->query_args` be updated too?

Reply:
> Skip the `isset( $query->query['term'])` just unset.

As I recall, I don't think the nuanced logic should ignore if that `term` is set. I think that would present another edge case, but I could be wrong... are you sure to remove it?

Reply:
> As I recall, I don't think the nuanced logic should ignore if that `term` is set. I think that would present another edge case, but I could be wrong... are you sure to remove it?

You're still checking if both `term` and `tax_query`: if both set, you can just unset.

Reply:
It's not a bug, though.

Reply:
Not exactly - it is on different vars. I think that was part of the specific scenario, those vars were somewhat missmatched and were an indicator that someone was not intentionally searching on that "additional" taxonomy. I was trying to rule out intentional search vs the mutated one done by `get_posts()`.